client/my-sites/email/email-providers-comparison/in-depth/email-provider-price.tsx

```diff
@@ -14,6 +17,9 @@ const EmailProviderPrice = ( {
 	selectedDomainName,
 }: EmailProviderPriceProps ): ReactElement => {
 	const selectedSite = useSelector( getSelectedSite );
+	const cartKey = useCartKey();
+	const shoppingCartManager = useShoppingCart( cartKey );
+	const isDomainAdded = hasDomainInCart( shoppingCartManager.responseCart, selectedDomainName );
```

Comment:
Given that we pass this variable to the `isDomainInCart` props below we should probably name it the same:
```suggestion
	const isDomainInCart = hasDomainInCart( shoppingCartManager.responseCart, selectedDomainName );
```